nanome_rmsd/RMSD.py

```diff
@@ -24,8 +24,7 @@ def start(self):
         self.compare_index = None
 
     def on_run(self):
-        menu = self.menu
-        menu.enabled = True
+        self._menu.enabled = True
```

Comment:
Does RMSDMenu have an `enabled` property? Or should this be `self._menu._menu.enabled`?